harvest/api/_base.py

```diff
@@ -62,10 +62,11 @@ def __init__(self, path: str = None):
         yml_file = Path(path)
         if not yml_file.is_file() and not self.create_secret(path):
             debugger.debug("Broker not initalized with account information.")
-            return
-        with open(path, "r") as stream:
-            self.config = yaml.safe_load(stream)
+        else:
+            with open(path, "r") as stream:
+                self.config = yaml.safe_load(stream)
```

Comment:
allow apis to be instantiated without the need for a file. Useful for the yahoo streamer.